pkg/webui/styles/main.styl

```diff
@@ -12,8 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700')
-@import url('https://fonts.googleapis.com/css?family=IBM+Plex+Mono')
+/* source-sans-pro-regular */
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../assets/fonts/source-sans-pro-regular.eot'); /* IE9 Compat Modes */
+  src: local('Source Sans Pro Regular'), local('SourceSansPro-Regular'),
+       url('../assets/fonts/source-sans-pro-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('../assets/fonts/source-sans-pro-regular.woff2') format('woff2'), /* Super Modern Browsers */
+       url('../assets/fonts/source-sans-pro-regular.woff') format('woff'), /* Modern Browsers */
+       url('../assets/fonts/source-sans-pro-regular.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('../assets/fonts/source-sans-pro-regular.svg#SourceSansPro') format('svg'); /* Legacy iOS */
+}
+/* source-sans-pro-600 */
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 600;
+  src: url('../assets/fonts/source-sans-pro-600.eot'); /* IE9 Compat Modes */
+  src: local('Source Sans Pro SemiBold'), local('SourceSansPro-SemiBold'),
+       url('../assets/fonts/source-sans-pro-600.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('../assets/fonts/source-sans-pro-600.woff2') format('woff2'), /* Super Modern Browsers */
+       url('../assets/fonts/source-sans-pro-600.woff') format('woff'), /* Modern Browsers */
+       url('../assets/fonts/source-sans-pro-600.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('../assets/fonts/source-sans-pro-600.svg#SourceSansPro') format('svg'); /* Legacy iOS */
+}
+/* source-sans-pro-700 */
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 700;
+  src: url('../assets/fonts/source-sans-pro-700.eot'); /* IE9 Compat Modes */
+  src: local('Source Sans Pro Bold'), local('SourceSansPro-Bold'),
+       url('../assets/fonts/source-sans-pro-700.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('../assets/fonts/source-sans-pro-700.woff2') format('woff2'), /* Super Modern Browsers */
+       url('../assets/fonts/source-sans-pro-700.woff') format('woff'), /* Modern Browsers */
+       url('../assets/fonts/source-sans-pro-700.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('../assets/fonts/source-sans-pro-700.svg#SourceSansPro') format('svg'); /* Legacy iOS */
+}
+/* ibm-plex-mono-regular */
+@font-face {
+  font-family: 'IBM Plex Mono';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../assets/fonts/ibm-plex-mono-regular.eot'); /* IE9 Compat Modes */
```

Comment:
@kschiffer do we need this?

Reply:
I guess since they would be lazy-loaded on demand, it's ok to keep them.